engine/runtime/src/main/java/org/enso/interpreter/runtime/data/Type.java

```diff
@@ -34,7 +34,7 @@ public final class Type implements EnsoObject {
   private final Type supertype;
   private final Type eigentype;
   private final Map<String, AtomConstructor> constructors;
-  private boolean hasProjectPrivateConstructors;
+  private boolean isProjectPrivate;
```

Comment:
We want the compiler to see this as a compilation constant. Please try to make it `final`.

Reply:
Done in 19092a916157077a4f11d7fcbaff37a2701033e6